app/models/qualifications.rb

```diff
@@ -0,0 +1,20 @@
+class Qualifications
+  def initialize(profpost_flag:, is_pgde:, is_fe:)
+    @profpost_flag = profpost_flag
+    @is_pgde = is_pgde
+    @is_fe = is_fe
+  end
+
+  def to_a
```

Comment:
@fofr is this more appropriate logic now?

Reply:
maybe we should just return 

```ruby
:qts
:pgce
:pgde
:pgce_qts
:pgde_qts
:blank # for invalids etc
```

and omit the array altogether...

Reply:
Not keen on the idea of going back to single values. QTS, PGCE and PGDE are completely independent things in real life – it will make changes harder later on if we treat them as a single thing.

Also the logic of deriving them can be simpler if we treat them separately.